### DISCUSSION TITLE
やって欲しいこと書きますよ

### DISCUSSION THREAD

**Discussion by Hamster-crab** [HamsterCrab/Science-PC_Club_Game#60] [Ideas]
[Screencast from 2024-09-02 15-38-49.webm](https://github.com/user-attachments/assets/d762695a-3c66-4c31-88ed-5fe800b7abc5)
↑ゲーム背景(ちょっと苦戦したけど) ※ GIMPで編集しろ ※ resources/Nomal-World/GameMap.xcf

戦闘時の自分のターンのとき、二人目のハムスターさんの攻撃とかの操作が終わったら相手のターン2が始まるようにして
※ src/Battle/battle.cpp

**Comment by Hamster-crab** [comment]
雑魚の戦闘はわいが作るから大丈夫
先にボスをアンケートで決めて(GitHub内で)
決まったら各ボスのBGM作って
BGMに沿って戦闘の弾幕を作るから

**Reply by RainbowPuiPuiMolcar** [reply]
BGMはフリーまたはAIに作らせていいですか？

**Reply by RainbowPuiPuiMolcar** [reply]
一番上のやつに質問ですGIMPというのはダウンロードするのでしょうか

**Reply by RainbowPuiPuiMolcar** [reply]
ダウンロードしてたら時間なくなりそうだ

**Reply by RainbowPuiPuiMolcar** [reply]
マップ作りのためにnkoから砂浜の素材と雪山の素材を追加しておきました

**Reply by Hamster-crab** [reply]
何でもかんでもAI使うね

**Reply by RainbowPuiPuiMolcar** [reply]
音楽に関してはしょうがないでしょ

**Comment by Hamster-crab** [comment]
あとplayerの詳細設定とハムスターの詳細設定をアクセスしやすいようにjson形式でかいて ※ どこで生まれて...などの設定は求めていません そんな考えは捨てて下さい 二度と拾わないで下さい 体力などです もちろんわかっていますよね

**Reply by RainbowPuiPuiMolcar** [reply]
AIを使って書いていいですか？

**Reply by RainbowPuiPuiMolcar** [reply]
やり方が分かりませんとりあえずマップ間に合いそうだったらやります

**Reply by Hamster-crab** [reply]
AI使うようなもんじゃないよ

**Reply by Hamster-crab** [reply]
あとjsonはunityのセーブとかでも使えるらしいから覚えといたほうがいいよ

**Reply by RainbowPuiPuiMolcar** [reply]
後で教えてクレメンス

**Reply by Hamster-crab** [reply]
後で教えますよ
